rust/log-capture/src/avro_schema.rs

```diff
@@ -125,22 +125,6 @@ pub const AVRO_SCHEMA: &str = r#"
         }
     }],
     "doc": "Map of custom timestamp-valued attributes."
-    },
```

Comment:
these were super old iceberg stuff, complete waste of bytes to duplicate this data (its already in the attributes map)